src/stavisky_lab_to_nwb/stavisky/staviskynwbconverter.py

```diff
@@ -28,3 +34,37 @@ def __init__(
         verbose: bool = True,
     ):
         super().__init__(source_data=source_data, verbose=verbose)
+
+    def run_conversion(
+        self,
+        nwbfile_path: Optional[str] = None,
+        nwbfile: Optional[NWBFile] = None,
+        metadata: Optional[dict] = None,
+        overwrite: bool = False,
+        conversion_options: Optional[dict] = None,
+    ) -> None:
+        # TODO: ideally we'd have an `alignment_options` arg or something like that
+        # instead of completely reimplementing `run_conversion`
+        if metadata is None:
+            metadata = self.get_metadata()
+
+        self.validate_metadata(metadata=metadata)
+
+        self.validate_conversion_options(conversion_options=conversion_options)
+
+        self.temporally_align_data_interfaces(metadata=metadata)
+
+        with make_or_load_nwbfile(
+            nwbfile_path=nwbfile_path,
+            nwbfile=nwbfile,
+            metadata=metadata,
+            overwrite=overwrite,
+            verbose=self.verbose,
+        ) as nwbfile_out:
+            self.add_to_nwbfile(nwbfile_out, metadata, conversion_options)
+
+    def temporally_align_data_interfaces(self, metadata=None):
+        if metadata is not None:
```

Comment:
Instead of using the metadata, what would you think about setting the session start time as an attribute during init (either passed as an argument or fetched automatically) then used here? Then there's no need to override `run_conversion`